src/app/modules/main/wallet_section/send/account_item.nim

```diff
@@ -83,3 +87,6 @@ QtObject:
   QtProperty[QVariant] currencyBalance:
     read = getCurrencyBalanceAsQVariant
     notify = currencyBalanceChanged
+
+  proc canSend*(self: AccountItem): bool =
+    return self.canSend
```

Comment:
@alaibe as I see the only logical context where you need info if we can send from a certain account or not is here in the account item context. 

So my proposal is, that you just need this:
```
  proc getCanBeSentFrom*(self: AccountItem): bool {.slot.} =
    return self.getOperability()==AccountFullyOperable or
      self.getOperability()==AccountPartiallyOperable
  QtProperty[string] canBeSentFrom:
    read = getCanBeSentFrom
    notify = operabilityChanged
```

Exposing this you will have information in the send popup whether to display this account in the dropdown list or not.

Reply:
Cause I thought you need it on the qml side.